ur_description/launch/view_ur10.launch

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<launch>
+  <include file="$(find ur_description)/launch/ur10_upload.launch"/>
+
+  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" >
+    <param name="use_gui" value="true"/>
+  </node>
+  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
+  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find ur_e_description)/etc/view_robot.rviz" required="true" />
```

Comment:
this is a `.launch` file in `ur_description`, but it uses the RViz config from `ur_e_description`, even though `ur_description` also has an RViz config now.

Was that intended?

Reply:
Additionally: could we place the RViz config in a `cfg` sub dir? Instead of the `etc`?

Reply:
I changed the files accordingly